src/01-functions.js

```diff
@@ -109,10 +127,17 @@ function averageThree() {}
  *  compareTypes("left", "right");
  *  //> true
  *
- *  compareTypes("left", 10);
+ *  compareTypes("left", 10)
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b)
+ {
+  if (typeof a == typeof b)  
+  return true
+  else (typeof a != typeof b) 
+   
+ return false
+}
```

Comment:
Since `typeof a == typeof b` returns a boolean that will be `true` when you want to return `true` and `false` when you want to return `false`, you can simply return the value it returns.

The equivalent with numbers would be this function:

``` javascript
function add(a, b) {
  const sum = a + b;
  if (sum === 1) {
    return 1;
  } else if (sum === 2) {
    return 2;
  } else if (sum === 3) {
    return 3;
  } else if (sum === 4) {
    return 4;
  } else // you get the idea
}
```

You could simply return `a + b` in this case, and you can simply return the `typeof` calculation in the same way. Try it!

`